syntax = "proto3";

package finger_tracking.proto;

message Rectangle{
  int32 x = 1;
  int32 y = 2;
  int32 width = 3;
  int32 height = 4;
}

message Key {
  string name = 1;
  Rectangle AABB = 2;
}

message KeyboardShape {
  repeated Key keys = 1;
}
